bitrise.yml

```
@@ -172,51 +125,35 @@ workflows:
 
   test-api:
     steps:
-    - script:
-        title: GOLIST_WITHOUT_VENDOR
-        inputs:
-        - content: |-
-            set -ex
-            no_vendor_paths="$(go list ./... | grep -v vendor)"
-            envman add --key GOLIST_WITHOUT_VENDOR --value "$no_vendor_paths"
-          opts:
-            # opts, for bitrise.yml parsing test in the workflow editor
-            is_expand: true
-            title: The Script Content - test title
-            is_required: true
-            is_dont_change_value: false
     - script:
         title: Go Test
         inputs:
         - content: |-
             #!/bin/bash
             set -ex
-            # go test $GOLIST_WITHOUT_VENDOR
             go test ./...
     - script:
         title: Go Vet
         inputs:
         - content: |-
             #!/bin/bash
             set -ex
-            go vet $GOLIST_WITHOUT_VENDOR
+            go vet
```

I think we need to add `./...` here, to do it for all packages.

```suggestion
            go vet ./...
```